test/taylorize.jl

```diff
@@ -1058,8 +1093,23 @@ import Logging: Warn
         end
         tT = t0 + Taylor1(_order)
         qT = [0.2, 0.0, 0.0, 3.0] .+ zero(tT)
-        @test_throws MethodError TI.__jetcoeffs!(
-            Val(true), kepler1!, tT, qT, similar(qT), similar(qT), -1.0)
+        parse_eqs, rv = (@test_logs (Warn, unable_to_parse(kepler1!)) TI._determine_parsing!(
+            true, kepler1!, tT, qT, similar(qT), -1.0))
+        @test !parse_eqs
+        @test_throws MethodError TI.__jetcoeffs!(Val(kepler1!), tT, qT, similar(qT), -1.0, rv)
+
+        # Error: `@taylorize` allows only to parse up tp 5-index arrays
+        ex = :(function err_arr_indx!(Dz, z, p, t)
+            n = size(z,1)
+            arr5 = Array{typeof(z[1])}(undef,n,1,1,1,1)
+            for i in eachindex(z)
+                arr5[i,1,1,1,1] = 0.0
+                Dz[i] = z[i] + arr5[i,1,1,1,1]
+            end
+            nothing
+        end)
+        @test_throws "Error: `@taylorize` allows only to parse up tp 5-index arrays" TI._make_parsed_jetcoeffs(ex)
```

Comment:
Maybe the error here can be fixed by changing this line with ``@test_throws ErrorException("Error: `@taylorize` allows only to parse up tp 5-index arrays") TI._make_parsed_jetcoeffs(ex)``?

Reply:
I'm checking this right now 😄; I'll push a change soon...